src/activate.ts

```diff
@@ -0,0 +1,64 @@
+import * as os from 'os'
+import * as osPath from 'path'
+import * as core from '@actions/core'
+import { executeGetOutput, pixiCmd } from './util'
+
+type ShellHook = {
+  environment_variables: Record<string, string>
+}
+
+const splitEnvironment = (shellHook: ShellHook): [Record<string, string>, string?] => {
+  if (os.platform() === 'win32') {
+    // On Windows, environment variables are case-insensitive but JSON isn't...
+    const pathEnvs = Object.keys(shellHook.environment_variables).filter((k) => k.toUpperCase() === 'PATH')
+    const caseSensitivePathName = pathEnvs[0]
```

Comment:
Will this throw an error or be undefined if pathenvs length === 0?

Reply:
I don't think it can happen that `PATH` is not set but I can check

Reply:
I'm not sure why this is done before the `if` statement, I just moved it below